### PR TITLE
Sous Chefs Adoption

### PR DESCRIPTION
**Pull request by EMH333**
# Description

Adopt into Sous Chefs

## Check List

- [x] All tests pass. See TESTING.md for details.
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ This file is used to list changes made in each version of the motd-tail cookbook
 
 ## Unreleased
 
-- resolved cookstyle error: metadata.rb:8:1 refactor: `ChefStyle/OverlyComplexSupportsDependsMetadata`
+- Cookstyle fixes
```
Comment (ramereth) [inline CHANGELOG.md:7]: Also add `Sous Chefs Adoption`

File: kitchen.dokken.yml [hunk 2]
```diff
@@ -15,55 +15,40 @@ verifier:
...
+    driver:
+      image: dokken/debian-10
+      pid_one_command: /bin/systemd
+      intermediate_instructions:
+        - RUN /usr/bin/apt-get update
+
+  - name: centos-7
+    driver:
+      image: dokken/centos-7
+      pid_one_command: /sbin/init
```
Comment (ramereth) [inline kitchen.dokken.yml:35]:
```suggestion
      pid_one_command: /usr/lib/systemd/systemd
```

File: metadata.rb [hunk 3]
```diff
@@ -1,13 +1,12 @@
...
+license           'Apache-2.0'
 description 'Updates motd.tail with Chef Roles'
-version '5.1.0'
+version           '5.1.0'
+source_url        'https://github.com/sous-chefs/motd-tail'
+issues_url        'https://github.com/sous-chefs/motd-tail/issues'
+chef_version      '>= 12.15'
 
 supports 'debian'
 supports 'ubuntu'
```
Comment (ramereth) [inline metadata.rb:12]: Please add all the RHEL/CentOS derivatives as well.

**Review by ramereth — CHANGES_REQUESTED** [review]

**Review by ramereth — APPROVED** [review]

**Comment by kitchen-porter** [comment]
Released as: [5.2.0](https://github.com/sous-chefs/motd-tail/releases/tag/5.2.0)

**Pull request merged**